packages/cli/locale-kit/locale-kit.ts

```diff
@@ -9,21 +9,9 @@ import {
     LOCALE_NAMES,
     readMessages,
     writeMessages,
+    findAllUnsyncedLocales,
```

Comment:
It would be better to remove `findAllUnsyncedLocales` in `utils.js`. Anyway `functon findAllUnsyncedLocales()` should be alongside with `function removeAllUnusedKeys()`

Reply:
https://github.com/DimensionDev/Maskbook/blob/f47876ec70a305c25f7fe17b5c197c2f8ce3a071/packages/cli/locale-kit/locale-kit.ts#L75-L85